README.md

```diff
@@ -29,6 +29,7 @@ This template comes with:
 
 1. Clone this repo
 1. Install PostgreSQL in case you don't have it
+1. Make the `bootstrap.sh` script executable: `chmod +x bootstrap.sh`
```

Comment:
what do you think about actually making the script executable and committing it to the repo?

Reply:
@santib than's actually a better idea, thanks! I've just made the change now 🙂